### ISSUE TITLE
Issues aren't loading from board view (Priority, 12/16/2015)

### ISSUE THREAD

**Issue by boydgreenfield** [ZenHubIO/support#517]
Issues never load from board view as of 12/16/2015 for everyone on our team:

![image](https://cloud.githubusercontent.com/assets/535969/11849362/5607e1d6-a3dc-11e5-8728-31914069486e.png)

JS console error:

```
Uncaught TypeError: e.match is not a functionc.extractIssueContentFromGHPage @ main.js:15s.Model.extend.handleSuccessRequest @ main.js:25(anonymous function) @ main.js:25h @ main.js:3d.fireWith @ main.js:3n @ main.js:4(anonymous function) @ main.js:4
```

**Comment by boydgreenfield** [comment]
Oops, this is a duplicate of #515 and #516. Sorry!

**Comment by azenMatt** [comment]
A fix is now live on v2.13.17 :smile:
